tracr/actions/thresh_henry.py

```diff
@@ -0,0 +1,95 @@
+
+# coding: utf-8
+
+# In[1]:
+
+import numpy as np
+import scipy as sp
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import math
+
+from scipy import misc
+from scipy import ndimage
+
+# Read in a 3D array
+im = np.load('A13_raw.npy')
```

Comment:
If this is test code, then it should go into tracr/test/test_threshold.py

With `nosetests`, this would look like:

```
import os, sys
sys.path.append('..')
import numpy as np
# do we really want a file named "thresh_henry"? Create a folder named "threshold" that contains a __init__.py
# and a file with this implementation named "trilevel_otsu.py"
#
# =======
# __init__.py
# -----------
# from trilevel_otsu import trilevel_otsu
# =======
#
from tracr.actions.thresh_henry import trilevel_otsu

class TestClass:
    def setUp(self):
        self.im = np.load('A13_raw.npy')

    def test_trilevel_otsu(self):
        # perform a threshold on an image you've already characterized,
        # let's say those levels are L and U (for lower and upper thresholds, respectively)
        thresholds = trilevel_otsu(self.im[500], bins=0)
        assert np.allcose(thresholds, (L, U)), \
            'Thresholds do not match expected levels: {} != {}'.format(thresholds, (L, U))
```